lib/encrypter.rb

```diff
@@ -17,7 +17,7 @@ def self.get_encryption(tokens, shifts)
 
   def group_tokens(tokens)
     grouped_tokens = []
-    tokens.each_slice(4) {|token_group| grouped_tokens << token_group}
+    tokens.each_slice(4) {|token| grouped_tokens << token}
     @terminal_tokens = grouped_tokens.pop if grouped_tokens.last.size < 4
```

Comment:
Following the style convention - an iterator's block variable is the one single element of a collection. "one token out of many tokens"